### DISCUSSION TITLE
Improving Mouse Interaction in Scale Original Mode for Enhanced Usability

### DISCUSSION THREAD

**Discussion by Seraphli** [rustdesk/rustdesk#6420] [Feature Request]
Currently, when using the Scale Original mode, it seems that the relationship between the mouse and the screen is determined by the distance of the mouse from the center of the client's screen. This is quite inconvenient. When using it, you need to move the mouse very close to the edge to see the information at the edge. Is it possible to keep the mouse centered? The screen would only move to the edge when the mouse is less than half a screen's distance from the boundary. Alternatively, a method similar to the screen-dragging feature in strategy games could be used, where the screen moves only when the client's mouse quickly moves to the edge. The current method is very inconvenient, especially when you need to view text at the bottom of the screen and simultaneously operate content in the middle of the screen, particularly when coding.

**Comment by Chocolateandmilkwin** [comment]
I would add that it would be nice if the area that moves the screen location was smaller than the full screen, it is quite annoying that you cannot see what is at the very edge of the screen unless you move the mouse to the the very edge, if you have multiple monitors it becomes almost impossible.
